Zend/zend_alloc_sizes.h

```diff
@@ -30,7 +30,7 @@
 
 /* num, size, count, pages */
 #define ZEND_MM_BINS_INFO(_, x, y) \
-	_( 0,    8,  512, 1, x, y) \
+	_( 0,    8,  256, 1, x, y) \
```

Comment:
Didn't remove the first bin, as this complexifies both branches of zend_mm_small_size_to_bin. Instead I clamp its size up to 16 in `_BIN_DATA_SIZE`.